jujugui/static/gui/src/app/assets/javascripts/jujulib/test-juju.js

```diff
@@ -5,14 +5,96 @@
 chai.config.includeStack = true;
 chai.config.truncateThreshold = 0;
 
-describe('juju.js', function() {
-
-    it('exists', function() {
-        var cs = new jujulib.charmstore();
-        var env = new jujulib.environment();
-        var identity = new jujulib.identity();
-        assert.isTrue(cs instanceof jujulib.charmstore);
-        assert.isTrue(env instanceof jujulib.environment);
-        assert.isTrue(identity instanceof jujulib.identity);
+describe('jujulib', function() {
+
+    describe('environment manager', function() {
+        var _makeXHRRequest = function(obj) {
+            return {target: {responseText: JSON.stringify(obj)}};
+        };
+
+        it('exists', function() {
+            var env = new window.jujulib.environment();
```

Comment:
can you have these instances available to an afterEach call and then set it to null after every test. I'm being extra careful so that we don't get cascading test failures in this suite :)